### PR TITLE
Move console script to bin folder. Fix Transifex issues.

### PR DESCRIPTION
**Pull request by anton-vlasenko**
**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/8322
| BC breaks? | `app/console` is no longer available. Use `bin/console` instead.
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
`app/console` script is still included into the distribution package of Mautic 3.
Also, there are Transifex related errors in the console when we try to run console commands.
E.g.
```
[WARNING] Some commands could not be registered:
In getTransifex_FactoryService.php line 8:                                                              
  Class 'RicardoFiorani\GuzzlePsr18Adapter\Client' not found                                                               

In getTransifex_FactoryService.php line 8:                                                              
  Class 'RicardoFiorani\GuzzlePsr18Adapter\Client' not found  
```

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Download and install Mautic  `3.0.0-alpha` from https://github.com/mautic/mautic/releases/tag/3.0.0-alpha.
2. Try to run a console command. You will get Transfex related errors in the console (see above).

#### Steps to test this PR:
1. Install Mautic from this branch: https://github.com/anton-vlasenko/mautic/commits/move-console-script-to-bin-folder (development version).
2. Add a `3.0.0-alpha` tag to the latest commit in that branch (don't push it! to the repository; this is needed for testing only. You have too add it using --force flag).
```bash
git tag -a 3.0.0-alpha <commit_hash> -m "3.0.0-alpha" --force
```
3. Make sure your diff.renamelimit value is high enough.
```bash
git config diff.renamelimit 1000000
```
4. Run `php build/package_release.php`.
5. Install Mautic using the package that you built on the previous step (3.0.0-alpha.zip).
6. Check that there is no app/console file in your new Mautic installation.
7. Try to run console commands. There must be no Transifex errors.

#### List backwards compatibility breaks:
1. `app/console` is no longer available. Use `bin/console` instead.

### CODE REVIEW

File: build/excludefiles.txt [hunk 1]
```diff
@@ -6,9 +6,21 @@
 .travis.yml
 *.git*
 *.idea*
+app/console
```
Comment (hluchas) [inline build/excludefiles.txt]: Delete of command above is solution. This line is not needed

File: build/excludefiles.txt [hunk 2]
```diff
@@ -6,9 +6,21 @@
...
 app/phpunit.xml.dist
+bin/codecept
```
Comment (hluchas) [inline build/excludefiles.txt]: not sure about this
What about to prevent access with .htaccess and keep this folder content?

Comment (hluchas) [reply build/excludefiles.txt]: Please use .htaccess only

Comment (anton-vlasenko) [reply build/excludefiles.txt]: Fixed in e1a9e8909db0be518b27befc1fb87e0bad724f65

File: build/excludefiles.txt (hunk 1 above)
Comment (anton-vlasenko) [reply build/excludefiles.txt]: I agree that the removing app/console is enough, but this line ensures that nobody will accidentally add it back to the repository.

**Comment by anton-vlasenko** [comment]
> When I try to use console:
> [console.log](https://github.com/mautic/mautic/files/4122127/console.log)
> 
> `Tests/DataFixtures/ORM` does not exist in build app, because test folders are not included, but class is required by some command via DI.
> 
> Soulution is to move this class to `bundles/SomeBundle/DataFixtures/..`

Yes, I agree. I've also thought about moving it to a different location.
@lukassykora Is working on a similar issue and if we fix it now there could be conflicts.
So we've decided to create a separate task for this.
https://github.com/mautic/mautic/issues/8359

**Comment by anton-vlasenko** [comment]
@escopecz Could you please merge this PR? Thank you in advance.

**Pull request merged**